Automattic/woocommerce-payments
# Fix JS error when clicking GPay button from blocks checkout with subscription in cart.

Fixes #8499

#### Changes proposed in this Pull Request
This PR passes along the click event to the `onButtonClick()` function when the GPay PRB button is clicked.


#### Testing instructions

1. Go to a JN store.
2. Create a subscription with a sign-up fee
3. Log out from the store
4. Add subscription to the cart, navigate to Blocks checkout
5. Click GPay button and see a browser alert that you need to log in or create an account on the site before you can complete the transaction.

You should see something like this after clicking the GPay button:
![screenshot_2024-04-04_at_11 47 38___am_720](https://github.com/Automattic/woocommerce-payments/assets/1558827/9c6a62dc-3a82-42fa-8db6-740b049f7393)

-------------------


**Post merge**
